src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1226,6 +1236,12 @@ public void onWorldTick(@NotNull final TickEvent.WorldTickEvent event)
                     spawnCitizen();
                 }
             }
+
+            if (event.world.getDifficulty() != EnumDifficulty.PEACEFUL && Configurations.doBarbariansSpawn && MobEventsUtils.isItTimeToRaid(event.world, this))
+            {
+                System.out.println("Stuff");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS106)